### PR TITLE
Add colored tutorial tags

### PR DESCRIPTION
**Pull request by samajammin**
## Description

**WIP** Add colored tutorial tags

## Related Issue

None

### CODE REVIEW

File: src/components/TutorialTags.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  let hash = 0
+  for (const char of string) {
+    const code = char.charCodeAt(0)
+    hash = (hash << 5) - hash + code
+    hash |= 0
+  }
+  return Math.abs(hash)
+}
+
+// TODO update to Theme variables
```
Comment (samajammin) [inline src/components/TutorialTags.js:17]: @ryancreatescopy ready for you to sub in the actual colors 🎨 

Please add theme dark & light mode in Theme.js

Comment (ryancreatescopy) [reply src/components/TutorialTags.js:17]: @samajammin not sure how to add theme values into this array

Comment (ryancreatescopy) [reply src/components/TutorialTags.js:17]: image

Comment (samajammin) [reply src/components/TutorialTags.js:17]: Looks good @ryancreatescopy! One thought - the 2nd green (mint?) looks very similar to the first green (at least for me), particular on dark mode. I wonder if we replace that with a purple or brown or some other more distinct color?

**Pull request merged**